packages/fxa-payments-server/src/components/Header/index.test.tsx

```diff
@@ -31,9 +31,6 @@ describe('Header', () => {
 
     const { queryByTestId } = subject();
 
-    const branding = queryByTestId('branding');
-    expect(branding).toHaveAttribute('title', 'firefox');
-
```

Comment:
Removed as both the title and the test aren't necessary for the logo (it contains the name in it already) and the alt tag takes care of both the absence of the image and being read aloud by a screen reader.